packaging/requirements.py

```diff
@@ -22,7 +22,7 @@
 
 
 if TYPE_CHECKING:  # pragma: no cover
-    from typing import List
+    from typing import List, Optional as TOptional, Set
```

Comment:
Why not keep this as `Optional`?

Reply:
Because `Optional` is imported from pyparsing on line 11.